### PR TITLE
Bump kicbase on head

### PR DESCRIPTION
**Pull request by medyagh**
building changes on HEAD

also fixed the image build process, that was adding an extra -snapshot to the image name

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -612,7 +611,7 @@ push-storage-provisioner-manifest: $(shell echo $(ALL_ARCH) | sed -e "s~[^ ]*~st
 	docker manifest push $(STORAGE_PROVISIONER_MANIFEST)
 
 .PHONY: push-docker
-push-docker: # Push docker image base on to IMAGE variable
+push-docker: # Push docker image base on to IMAGE variable (used internaly by other targets)
```
Comment (tstromberg) [inline Makefile:614]: "internally"

**Review by tstromberg — CHANGES_REQUESTED** [review]
Please split the Makefile & code change, In the unlikely case that this PR needs to reverted. Thanks!

**Review by sharifelgamal — APPROVED** [review]

**Pull request merged**